src/index.html

```diff
@@ -7,6 +7,340 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="images/logo.png"
+          alt="logo"
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" 
+              href="#"
+            >Apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
Go through the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) more thoroughly. Try to fix it everywhere
```suggestion
          <li class="nav__item">
            <a 
              class="nav__link is-active" 
              href="#"
            >
              Apple
            </a>
          </li>
          
          <li class="nav__item">
```

Reply:
Not fixed

Reply:
not fixed